configs/_base_/tta/centernet_tta.py

```diff
@@ -0,0 +1,39 @@
+tta_model = dict(
+    type='DetTTAModel',
+    tta_cfg=dict(nms=dict(type='nms', iou_threshold=0.5), max_per_img=100))
+
```

Comment:
It would be better to place the model-related TTA files in the model's config folder.